### PR TITLE
TEN-1452/24.10.0/home directory permissions tests

### PR DESCRIPTION
**Pull request by markbainix**
Ten 1452 home directory permissions
image

### CODE REVIEW

File: test_cases/webui/local_users/test_local_users.py [hunk 1]
```diff
@@ -314,3 +314,238 @@ def verify_user_login_after_edit(self, users) -> None:
...
+        Test Steps:
+        1. Edit home directory user permission fields
+        2. Click Save button
+        3. Verify home directory user permission fields are set
+        4. Verify home directory user permission are applied
+        """
+        API_POST.start_service('ssh')
+        assert LU.is_user_visible(users['username']) is True
+
+        # LU.unset_show_builtin_users_toggle()
```
Comment (jordan-mcgillivray005) [inline test_cases/webui/local_users/test_local_users.py:333]: This should probably be uncommented

File: test_cases/webui/local_users/test_local_users.py [hunk 2]
```diff
@@ -314,3 +314,238 @@ def verify_user_login_after_edit(self, users) -> None:
...
+        API_DELETE.delete_user('groupuser')
+
+    @allure.tag("Update")
+    @allure.story("Verify Local Users - Home Directory Permissions-Other")
+    def test_verify_home_directory_permissions_other(self, users) -> None:
+        """
+        Summary: This test verifies home directory permissions for other are applied
+
+        Test Steps:
+        1. Add group user
```
Comment (jordan-mcgillivray005) [inline test_cases/webui/local_users/test_local_users.py:479]: this should say other user

File: test_cases/webui/local_users/test_local_users.py (hunk 1 above)
Comment (markbainix) [reply test_cases/webui/local_users/test_local_users.py:333]: not really needed. earlier test in file unsets.

**Review by jordan-mcgillivray005 — APPROVED** [review]

File: test_cases/webui/local_users/test_local_users.py (hunk 2 above)
Comment (markbainix) [reply test_cases/webui/local_users/test_local_users.py]: fixed

**Pull request merged**